### PR TITLE
Add DAG parsing tests

### LINKED ISSUE #20: DAG parsing tests
We should add DAG parsing tests. We can use this as an example: 

https://github.com/WordPress/openverse-catalog/blob/dad3cb49618f38aadb3ae7772f98ca77180bf35c/tests/dags/test_dag_parsing.py

### PR DESCRIPTION
**Pull request by AetherUnbound**
Resolves #20

**Note**: This should be rebased and merged _after_ all other open PRs are merged.

### CODE REVIEW

**Comment by mpuckett159** [comment]
Looks fine too but don't have the brain space for this right now sorry :(

File: tests/dags/test_dag_parsing.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+
+def test_dag_import_errors():
+    # Attempt to load all DAGs in the DAG_FOLDER, and detect import errors
+    dagbag = DagBag(dag_folder=DAG_FOLDER, include_examples=False)
+
+    dag_errors = []
+    for filename in dagbag.import_errors.keys():
+        dag_errors.append(Path(filename).name)
```
Comment (sea-kelp) [inline tests/dags/test_dag_parsing.py:30]: Super minor comment but I think this could be a dict comprehension?

**Review by sea-kelp — APPROVED** [review]
I think this makes sense to me!

File: tests/dags/test_dag_parsing.py (hunk 1 above)
Comment (AetherUnbound) [reply tests/dags/test_dag_parsing.py]: You're right, fixed!

**Pull request merged**